### DISCUSSION TITLE
Add node directory to .eslintignore

### DISCUSSION THREAD

**Discussion by dvbeek** [quasarframework/quasar#17614] [Ideas / Proposals] [answered]
Is it possible to add the `/node` directory to the lint ignore list?

Our front-end projects always have the node directory in the project, using maven with the eirslett plugin. 
So after creating projects we always end up with thousands of linting problems, because it will also validate the node directory.

**Comment by rstoenescu** [comment, marked as answer]
Hi,

You can add it yourself, but I wouldn't add it to the create-quasar package (that spawns new Quasar projects) because some of the users might have this folder for other purposes, which would lead to unnecessary friction.

**Reply by dvbeek** [reply]
Hi Razvan,

Thanks for your response.

I don't understand in what scenario it would be important to perform linting on the root node directory, when creating a new project...
In my opinion it would be more important to prevent problems in creating a project (which can cause new users to loose hours), than to make sure linting is performed in the root node, for the weird case that anybody would ever place source files here....

But this is your own choice, of course, so I will have to find another way for our company to solve this 😊

**Reply by rstoenescu** [reply]
It's a moot point, because:

1. As long as any file or folder are not Quasar specific or part of the recommended environment, there shouldn't be a config for them in the spawned project folders. The customization is left for the user to do.
2. You are requesting a change into the project folders that get spawned by the "yarn/pnpm/.. create quasar" command. This happens only ONCE per project. Once the project is spawned, it's out of the control of the create command. You already have a project, so add the folder yourself to the ignore list. It makes no sense to say "I will have to find another way". What am I missing?

**Reply by dvbeek** [reply]
Yes, this only a problem for creating new projects (which we do a lot).

I have to find a way for us to create projects, without (me or my colleagues) loosing time on the problems that always occur in this situation.
This means documenting or automating that after creating the project, we always add this row in the .eslintignore.

I think there is a way to configure the node directory in the eirslett frontend-maven-plugin, which I can probably use.

Thanks for your replies 👍

**Reply by rstoenescu** [reply]
If you automate things, you might want to look into an App Extension https://quasar.dev/app-extensions/introduction
This is exactly what it's for.
